### ISSUE TITLE
Question about the pydantic version

### ISSUE THREAD

**Issue by ashpreetbedi** [FlipsideCrypto/sdk#31]
Hi flipside team,

Wondering if there's any planned update to the pydantic version pinned in the python sdk?
Its currently at v1 which is incompatible with every other python library because they depend on pydantic v2

Thank you!

**Comment by Kempasso** [comment]
I agree! Due to the outdated version of Pydantic, using your SDK becomes practically impossible. I have to rely on the API and its two-step data retrieval strategy, which is very inconvenient.

**Comment by simonasmulevicius-humbility** [comment]
Totally agree! Flipside should upgrade their pydantic version 🙏
